scheduler/src/cook/mesos/api.clj

```diff
@@ -1385,7 +1409,9 @@
              :handler (read-jobs-handler conn fid task-constraints gpu-enabled? is-authorized-fn)}
        :post {:summary "Schedules one or more jobs."
               :parameters {:body-params RawSchedulerRequest}
-              :responses {200 {:schema [JobResponse]}}
+              :responses {201 {:description "The jobs were successfully scheduled."}
+                          400 {:description "One or more of the jobs were incorrectly specified."}
+                          409 {:description "One or more of the jobs UUIDs are already in use."}}
```

Comment:
@wyegelwel This PR removes the possibility of a 422.  422 was really being used when 500 would have been more appropriate (it was only used to report that an unknown error occurred, it was never indicating a specific correctable problem with the content of the request).   

500 could theoretically occur on basically ANY endpoint depending on where an unpredictable error occurs...  I can add that here but it really applies to the whole API (and basically any HTTP API).